tracing-log/src/lib.rs

```diff
@@ -221,6 +221,27 @@ impl AsTrace for log::Level {
 
 /// A simple "logger" that converts all log records into `tracing` `Event`s,
 /// with an optional level filter.
+///
+/// Be careful to set appropriate log levels:
```

Comment:
We should maybe note something about it being preferable in most cases to not filter logs on the `LogTracer`, but instead let the tracing `Subscriber` be responsible for filtering log events. The level filter should generally only be used when we know in advance we want to ignore all logs of a certain verbosity before they even make it to the `Subscriber`.

Perhaps the default should be `log::LevelFilter::Trace`, so that no log records are disabled...

Reply:
Having to set 3 different filter (log, logtracer and subscriber) feels overcomplicated. Which use case does `LogTracer`'s filter have? I wonder if it not just duplicating `log::set_max_level`'s behavior and could be removed to simplify this.

Reply:
We should probably remove the filter field from `LogTracer`.